.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,68 @@
+name: Brazillian Orders CI/CD Pipeline to EC2 
+
+on:
+  push:
+    branches: [ "orders_api_deployment" ]
+  pull_request:
+    branches: [ "orders_api_deployment" ]
+
+jobs:
+  build_and_push_image:
+    name: Login, Build and Push Docker Image
+    runs-on: ubuntu-latest
+    environment: development
+    steps:
+    - name: Checkout the code
+      uses: actions/checkout@v4
+    - name: Login Dockerhub
+      uses: docker/login-action@v1
+      with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+    - name: remove existing image if it exists
+      run: docker image rm savepeter2/brazillian-orders-api-deployment:latest || true
+    - name: Build the Docker Image
+      # run: docker build -t savepeter2/brazillian_orders_deployment:latest .
+      run: docker compose build
+    - name: Tag the image
+      run: docker tag brazillian-orders-api-deployment savepeter2/brazillian-orders-api-deployment:latest
+    - name: Push to Docker hub
+      run: docker push savepeter2/brazillian-orders-api-deployment:latest
+
+  deploy_to_ec2:
+    name: Pull image from Dockerhub and deploy to EC2
+    needs: build_and_push_image
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Deploy to EC2
+      env:
+        PRIVATE_KEY: ${{ secrets.EC2_SSH_PRIVATE_KEY }}
+        HOST: ${{ secrets.EC2_HOST }}
+        USER: ${{ secrets.EC2_USER }}
+      run: |
+        echo "$PRIVATE_KEY" > private_key && chmod 400 private_key
+            ssh -o StrictHostKeyChecking=no -i private_key ${USER}@${HOST} '
+              sudo su - root << EOF
+                sudo docker pull savepeter2/brazillian-orders-api-deployment:latest
+                sudo docker rm -f api-deployment-container || true
+                sudo docker run -d -p 8080:8000 \
```

Comment:
Why aren't you specifying the arguments via docker-compose file or Dockerfile?

Reply:
I'll fix that I think specifying the arguments via docker-compose will provide a consistent configuration between the development and deployment

Reply:
I don't really have a specific reason for not doing that

Reply:
Alright